### PR TITLE
Roles Button Dropdown

### LINKED ISSUE #160: [BUG] Access Management: Roles Button + Dropdown Misaligned
## Description

The buttons for admin/creator/volunteer look incorrectly aligned. 
The drop-down when selecting a new role for an email has a background that is bigger than the buttons for some reason.

### PR DESCRIPTION
**Pull request by uma-anand**
## Roles Button Dropdown

Issue Number(s): #160 .

What does this PR change and why?

### Checklist

- [ ] Requirements have been implemented
- [ ] Acceptance criteria are met
- [ ] Database schema docs have been updated or are not necessary
- [ ] Code follows design and style guidelines
- [ ] Commits follow guidelines (concise, squashed, etc)
- [x] Relevant reviewers (Senior Dev/EM/Designers) have been assigned to this PR

### Critical Changes

- Database change/migration to run
- Environment config change
- Breaking API change

### Related PRs

- #number_of_pr

### Testing

Enumerate steps to test the functionality of your ticket. This should include edge cases and testing of error handling, if applicable.

### CODE REVIEW

File: web/components/AccessManagement/RoleSelector.tsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ function RoleSelector(props: PropertyType) {
       onOpen={onOpen}
       onClose={onClose}
       placement="bottom"
-      offset={[42, 0]}
+      offset={[0, 0]}
```
Comment (katsukixyz) [inline web/components/AccessManagement/RoleSelector.tsx]: Attribute is the same for both breakpoints, so just assign `offset=0`

**Review by katsukixyz — CHANGES_REQUESTED** [review]

File: web/components/AccessManagement/RoleSelector.tsx (hunk 1 above)
Comment (uma-anand) [reply web/components/AccessManagement/RoleSelector.tsx]: It doesn't let you have a one parameter offset, but because the default is 0 anyway, I just removed that attribute. Let me know if you want me to do something else instead.

**Comment by katsukixyz** [comment]
Resolved by #275
